api/lib/infrastructure/utils/normalize-non-breaking-space.js

```diff
@@ -1,9 +1,9 @@
 export function normalizeNonBreakingSpace(str) {
   if (!str) return str;
   return str
-    .replaceAll(/ ?(°C)/g, ' $1')
-    .replaceAll(/ ([;?!€$%])/g, ' $1')
-    .replaceAll(/ (:)/g, ' $1')
-    .replaceAll(/ ?(»)/g, ' $1')
-    .replaceAll(/(«) ?/g, '$1 ');
+    .replaceAll(/\p{Zs}?(°C)/gu, ' $1')
+    .replaceAll(/\p{Zs}([;?!€$%])/gu, ' $1')
+    .replaceAll(/\p{Zs}(:)/gu, ' $1')
+    .replaceAll(/\p{Zs}?(»)/gu, ' $1')
+    .replaceAll(/(«)\p{Zs}?/gu, '$1 ');
```

Comment:
```suggestion
    .replaceAll(/(«)\p{Zs}/gu, '$1 ');
```
Si j'ai bien compris la demande de Julie ... ^^